triage/labels.rst

```diff
@@ -26,6 +26,8 @@ These labels are used to specify the type of issue:
 * :gh-label:`type-crash`: for hard crashes of the interpreter, possibly with a
   core dump.
 * :gh-label:`type-feature`: for feature requests or enhancements.
+  Feature requests do not need :ref:`Version labels`;
```

Comment:
Avoid the capital V in the middle of a sentence:

```suggestion
  Feature requests do not need :ref:`version labels <Version labels>`;
```

Reply:
Good catch! BTW; I found the capitalised anchor names weird. Is this a new thing? I can't remember seeing it in the docs before.

Reply:
Hmm, I'm not sure, let's ask @AA-Turner.

Reply:
Well, if it is ok for Adam, I'd like to add the anchor in lower-case 🙂